server/main.lua

```diff
@@ -1,25 +1,22 @@
----@class vehicleData
----@field model string
----@field plate string
----@field mods string
-
----@class playerData
+---@class CreateEntityQuery
 ---@field license string
 ---@field citizenId string
-
+---@field model string
+---@field mods table
+---@field plate string
+---@field state number
```

Comment:
mark optional fields with ? and provide a description for things like mods and state. State should actually be an enum instead of a number type. For any that have a default value, state the default in the spec there as well.

Reply:
I dont think any of these fields are optional. When creating a vehicle entity, all these parameters should be filled in. As for the enum, idk how you wanna do that.

Reply:
Idk how you wanna do the enum for this